src/components/Firebase/firebase.js

```diff
@@ -1,6 +1,7 @@
 import app from 'firebase/app';
 import 'firebase/auth';
 import 'firebase/database';
+import 'firebase/firestore';
```

Comment:
Does the `npm install firebase` already come with Firestore? Wow, they really put everything in there! :D

Reply:
Yes and `storage` too)